packages/dose3/dose3.5.0.1-1/files/0006-Grep-carefully-for-ocaml-system.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/configure.ac b/configure.ac
+index d73d724..3653943 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -208,7 +208,7 @@ if test "$HAS_BENCHMARK" = "yes" ; then
+ fi
+ 
+ AS_IF([test "$OCAMLBEST" = "opt"],[OCAMLBESTCC=$OCAMLOPT],[OCAMLBESTCC=$OCAMLC])
+-OCAML_CC="$($OCAMLBESTCC -config | fgrep native_c_compiler | sed -e "s/native_c_compiler: \(.*\) .*/\1/")"
++OCAML_CC="$($OCAMLBESTCC -config | grep -P "^native_c_compiler:" | sed -e "s/native_c_compiler: \(.*\) .*/\1/")"
```

Comment:
The manpage for `grep -P` states that it is an experimental feature. Why is this needed? I don't see any perl-specific regexp here

Reply:
Also `grep -P` is not portable. Busybox's grep doesn't have it for example.

Reply:
This is also breaking BSD which doesn't have it either. I'm reverting this..